Configure

```diff
@@ -286,7 +286,14 @@ collect_information(
     qr/\s*(\w+)\s*=\s*(.*?)\s*$/ =>
         sub {
             # Only define it if there is a value at all
-            $version{uc $1} = $2 if $2 ne '';
+            if ($2 ne '') {
+                my $k = $1;
+                my $v = $2;
+                # Some values are quoted.  Trim the quotes
+                $v =~ s|^"||;
+                $v =~ s|"$||;
```

Comment:
I think this is more clear (and more correct, requiring balanced quotes)
```
$v = $1 if $v =~ /"(.*)"/;
```

Reply:
Good point.

Reply:
We are clearly in a Different World, if I'm making Perl suggestions to you. :)

Reply:
Hahaha, I was thinking the same, like "Wait what?  Did he just....?"